src/ad-targeting-youtube.ts

```diff
@@ -14,22 +12,14 @@ import type {
 
 const disabledAds: AdsConfigDisabled = { disableAds: true };
 
-const buildCustomParamsFromCookies = (): CustomParams =>
-	canUseDom()
-		? {
-				permutive: getPermutivePFPSegments(),
-				si: getCookie({ name: 'GU_U' }) ? 't' : 'f',
-		  }
-		: {};
-
 const buildAdsConfig = (
 	cmpConsent: ConsentState,
 	adUnit: string,
 	customParams: CustomParams,
 ): AdsConfig => {
 	const mergedCustomParams = {
 		...customParams,
-		...buildCustomParamsFromCookies(),
+		...buildPageTargeting(cmpConsent, false, true),
```

Comment:
Minor organisational if you agree - could we move this file and its test into `/targeting` pls and rename it to `youtube` to align with the other files.